newm-server/src/main/kotlin/io/newm/server/features/song/SongRoutes.kt

```diff
@@ -92,9 +101,48 @@ fun Routing.createSongRoutes() {
                         )
                     }
                 }
+                get("mintPayment") {
+                    with(call) {
+                        // Return the cbor value to pass to api.getUtxos(value) for proper utxo selection.
+                        // This is essentially the server telling the web how much to charge the user.
+                        respond(
+                            MintPaymentResponse(
+                                // FIXME: Don't hardcode the price to mint
```

Comment:
Is this fix reminder intended to be done before merge or later on another pr?

Reply:
This will be in a later PR. We have yet to figure out where we want store changeable config things like minting prices.